lego-webapp/components/ShareButton/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Flex, Icon } from '@webkom/lego-bricks';
+import { ShareIcon } from 'lucide-react';
+import { useAppDispatch } from 'lego-webapp/redux/hooks';
+import { addToast } from '~/redux/slices/toasts';
+
+type Props = {
+  title: string;
+  url: string;
+};
+
+const ShareButton: React.FC<Props> = ({ title, url }) => {
+  const dispatch = useAppDispatch();
+  const handleShare = async () => {
+    if (navigator.share) {
+      try {
+        await navigator.share({ title, url });
+      } catch (error) {
+        console.error('Kunne ikke dele:', error);
+      }
```

Comment:
Perhaps look into the [`canShare`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/canShare) method (as used in the previous `ShareButton` component that was removed)